src/ARTED/control/control_ms.f90

```diff
@@ -104,7 +107,7 @@ subroutine tddft_maxwell_ms
   write(file_energy_transfer, "(A,'energy-transfer.out')") trim(directory)
   write(file_ac_vac, "(A,'Ac_Vac.out')") trim(directory)
   write(file_ac_vac_back, "(A,'Ac_Vac_back.out')") trim(directory)
-  write(file_ac_m, "(A,'Ac_M',I6.6,'.out')") trim(process_directory), NXY_s
+  write(file_ac_ms, "(A,'Ac_m',I6.6,'.out')") trim(process_directory), imacro
```

Comment:
Here, 'imacro' becomes 'nmacro_e+1' because of the above loop at line 87. But, I think this behavior is not expected. Please check it.